app/controllers/issue/message_threads_controller.rb

```diff
@@ -17,6 +17,24 @@ def new
     @message = @thread.messages.build
     @message.body = issue.description if issue.threads.count == 0
     @available_groups = current_user.groups
+    @external_services = ExternalService.all
+  end
+
+  helper_method :new_external
+
+  def new_external
+    @thread = issue.threads.build
+    set_page_title nil, issue: issue.title
+    if current_group
+      @thread.group = current_group
+      @thread.privacy = current_group.default_thread_privacy
+    end
+    @message = @thread.messages.build
+    @message.body = issue.description
+    @thread.title = issue.title
```

Comment:
I'm not sure about setting the thread title equal to the issue title.
I think it should be:
Issue: New housing will be built
Thread: Increased congestion on the road
Thread: Cycle Parking on the New housing application is poor

etc.

See https://github.com/cyclestreets/cyclescape/issues/43 for context.